runtime/lua/vim/treesitter.lua

```diff
@@ -306,7 +306,7 @@ end
 
 --- Returns a list of highlight capture names under the cursor
 ---
----@param winnr (integer|nil) Window handle or 0 for current window (default)
+---@param winnr (integer|nil): |window-ID| or 0 for current window (default)
```

Comment:
FYI: `#` also should work, if tooling is getting confused. I didn't know that `:` could be used like this.

Reply:
Both `:` and `#` should be equivalent.

Reply:
Yeah, I got lsp warnings without `:`. I used : because it was used elsewhere in this file.